<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>497</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineedit_search_string"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_search">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../QtQuestWindow/QtQuestWindow/Resource.qrc">
            <normaloff>:/images/search.ico</normaloff>:/images/search.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_empty">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../QtQuestWindow/QtQuestWindow/Resource.qrc">
            <normaloff>:/images/sync.ico</normaloff>:/images/sync.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddButton">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>新建</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../QtQuestWindow/QtQuestWindow/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
